addons/website_slides_forum/views/forum_views.xml

```diff
@@ -22,6 +22,20 @@
         </field>
     </record>
 
+      <record id="view_forum_forum_list" model="ir.ui.view">
+        <field name="name">forum.forum.view.list.inherit.website_slides</field>
+        <field name="model">forum.forum</field>
+        <field name="inherit_id" ref="website_forum.view_forum_forum_list"/>
+        <field name="arch" type="xml">
+            <field name="website_id" position="after">
+                <field name="slide_channel_id"/>
```

Comment:
Shouldn't it be optional hidden ? Not sure this is something you always want to see in your forum list view (maybe to check with LNA)